python/python/glide/async_commands/server_modules/json.py

```diff
@@ -245,6 +245,68 @@ async def clear(
     return cast(int, await client.custom_command(args))
 
 
+async def debug(
+    client: TGlideClient,
+    subcommand: str,
```

Comment:
use enum there so user can't pass wrong argument

Reply:
As an option you can split this into 2 APIs: `debug_memory` and `debug_fields`, so description of the return value will be less confusing.

Reply:
Done the check for passing wrong arguments inside the function. 
+ Split the debug function into 2 APIs as per your suggestion. less confusing and more clean